Добавляет рецепт «Как создать организацию на GitHub» (#4736)
doka-guide/content: recipes/github-new-organization/index.md

---
title: "Как создать организацию на GitHub"

> description: "Работайте в команде на GitHub удобно с помощью организации."

Reviewer: ```suggestion
description: "Работать в команде на GitHub удобнее с помощью организации."
```

authors:
  - igsekor
related:
  - tools/version-control
  - tools/git-cli
  - tools/github-actions
tags:
  - article
---

## Задача

У вас есть команда для работы над проектами, надо организовать её работу на GitHub.

## Готовое решение

> Есть два пути. Самый короткий — вверху страницы нажать на кнопку «+» и в разворачивающимся списке выбрать пункт «New organization».

Reviewer: ```suggestion
Есть два пути. Самый короткий — в меню рядом со строкой поиска нажать на кнопку «Create something new», которая выглядит как знак плюс «+», и в открывшемся списке выбрать пункт «New organization».
```

> ![](images/top-user-menu.png.png)
> ![](images/new-organization-by-plus.png.png)

Reviewer: ```suggestion
![Меню GitHub.](images/top-user-menu.png)
![Открытое меню с опциями, где можно выбрать организацию.](images/new-organization-by-plus.png)
```

> Второй путь — зайти в меню профиля, выбрать пункт «Your organizations» и на открывшейся странице нажать кнопку «New organization».

Reviewer: ```suggestion
Второй путь — зайти в настройки вашего профиля и в разделе «Access» выбрать вкладку «Your organizations». В открывшемся содержимом вкладки надо перейти по ссылке «New organization». Она находится рядом с заголовком «Organizations».
```

> ![](images/your-organizations.png)

Reviewer: ```suggestion
![Меню в профиле пользователя GitHub.](images/your-organizations.png)
```

> ![](images/new-organization-by-page.png)

Reviewer: ```suggestion
![Кнопка «New organization».](images/new-organization-by-page.png)
```

> На открывшейся странице вам будет предложено выбрать план обслуживания вашей организации. Самый первый план бесплатный, он не даёт пользоваться некоторыми возможностями, но для базовой работы в принципе они, скорее всего, и не потребуются.

Reviewer: ```suggestion
На открывшейся предложат выбрать план обслуживания организации. Первый план бесплатный, он не даёт пользоваться некоторыми возможностями, но для базовой работы они, скорее всего, и не потребуются.

План можно выбрать на месяц или год. Другие два плана платные. Это «Team» для небольших команд и «Enterprise» для больших компаний.
```

> ![](images/choose-your-plan.png.png)

Reviewer: ```suggestion
![Планы для организаций.](images/choose-your-plan.png)
```

> После нажатия на соответствующую кнопку (например, «Create a free organization»), откроется новая страница, на которой можно задать уникальное имя «Organization account name
> » (аналог профиля) и контактная электронная почта «Contact email». Также необходимо выбрать, кому будет принадлежать организация «This organization belongs to». Доступны две опции:

Reviewer: ```suggestion
После того, как перешли по ссылке с планом, например, «Create a free organization», снова откроется новая страница. Задайте уникальное имя «Organization account name» (аналог профиля) и добавьте электронную почту «Contact email». Также выберите, кому будет принадлежать организация «This organization belongs to». Доступны две опции:
```

> - «My personal account» — если организация на GitHub не будет ассоциироваться с какой-то компанией или образовательной организацией, то смело выбирайте этот пункт.
> - «A business or institution» — а если будет — то этот.

Reviewer: ```suggestion
- «My personal account» — если организация на GitHub не будет ассоциироваться с компанией или образовательной организацией, смело выбирайте этот пункт.
- «A business or institution» — если будет ассоциироваться, выберите этот пункт.
```

> ![](images/setup-organization-first-step.png)

Reviewer: ```suggestion
![Страница с настройками организации.](images/setup-organization-first-step.png)
```

> Остаётся согласиться с условиями с помощью галочки и можно жать на кнопку «Next». На следующей странице можно выбрать и подключить членов команды. После этого можно нажать кнопку «Complete setup». Членов команды можно добавить и позже, тогда нужно нажать на ссылку «Skip this step».

Reviewer: ```suggestion
Остаётся согласиться с условиями с помощью чекбокса рядом с согласием на условия использования сервиса и можно жать на кнопку «Next» под формой. На следующей можете выбрать и подключить членов команды. Для этого в поле «Search by username, full name or email address» введите ник, имя или почту нужного участника. После нажмите на кнопку «Complete setup» под формой.

Членов команды можно добавить и позже, тогда нужно нажать на ссылку «Skip this step», которая находится рядом с «Complete setup».
```

> ![](images/setup-organization-second-step.png)

Reviewer: ```suggestion
![Выбор членов организации.](images/setup-organization-second-step.png)
```

> Организация создана, и вы можете настроить её под свои задачи. Для быстрой настройки прямо на открывшейся странице можно пройти по пунктам специального гайда для настройки всех основных моментов. Внизу страницы можно уже создавать репозитории в рамках организации кнопкой «Create new repository», также можно импортировать репозиторий из аккаунта или другой организации.

Reviewer: ```suggestion
Организация создана, и вы можете настраивать её под свои задачи и дальше. Для быстрой настройки прямо на открывшейся странице можно пройти по пунктам специального руководства для настройки всех основных моментов. Например, пригласить ещё участников, настроить уровни их доступа, создать пулреквест, защитить ветки проекта, настроить CI/CD и узнать больше о других возможностях GitHub.

В боковом меню найдёте настройки видимости проекта, ссылки на создание README-файла, дискуссии, а также можете создавать репозитории в рамках организации кнопкой «Create new repository» или импортировать репозиторий из аккаунта или другой организации.
```

> ![](images/new-organization-page.png)

Reviewer: ```suggestion
![Страница со всеми настройками организации.](images/new-organization-page.png)
```

---
title: "Как создать организацию на GitHub"
description: "Работать в команде на GitHub удобнее с помощью организации."
authors:
  - igsekor
related:
  - tools/version-control
  - tools/git-cli
  - tools/github-actions
tags:
  - article
---

## Задача

У вас есть команда для работы над проектами, надо организовать её работу на GitHub.

## Готовое решение

Есть два пути. Самый короткий — в меню рядом со строкой поиска нажать на кнопку «Create something new», которая выглядит как знак плюс «+», и в открывшемся списке выбрать пункт «New organization».

![Меню GitHub.](images/top-user-menu.png)
![Открытое меню с опциями, где можно выбрать организацию.](images/new-organization-by-plus.png)

Второй путь — зайти в настройки вашего профиля и в разделе «Access» выбрать вкладку «Your organizations». В открывшемся содержимом вкладки надо перейти по ссылке «New organization». Она находится рядом с заголовком «Organizations».

![Меню в профиле пользователя GitHub.](images/your-organizations.png)
![Кнопка «New organization».](images/new-organization-by-page.png)

> На открывшейся предложат выбрать план обслуживания организации. Первый план бесплатный, он не даёт пользоваться некоторыми возможностями, но для базовой работы они, скорее всего, и не потребуются.

Reviewer 2: ```suggestion
На открывшейся вкладке вам предложат выбрать план обслуживания организации. Первый план бесплатный, он не даёт пользоваться некоторыми возможностями, но для базовой работы они, скорее всего, и не потребуются.
```

План можно выбрать на месяц или год. Другие два плана платные. Это «Team» для небольших команд и «Enterprise» для больших компаний.

![Планы для организаций.](images/choose-your-plan.png)

> После того, как перешли по ссылке с планом, например, «Create a free organization», снова откроется новая страница. Задайте уникальное имя «Organization account name» (аналог профиля) и добавьте электронную почту «Contact email». Также выберите, кому будет принадлежать организация «This organization belongs to». Доступны две опции:

Reviewer 2: ```suggestion
После перехода по ссылке с планом, например, «Create a free organization», снова откроется новая страница. Задайте уникальное имя «Organization account name» (аналог профиля) и добавьте электронную почту «Contact email». Также выберите, кому будет принадлежать организация «This organization belongs to». Доступны две опции:
```

- «My personal account» — если организация на GitHub не будет ассоциироваться с компанией или образовательной организацией, смело выбирайте этот пункт.
- «A business or institution» — если будет ассоциироваться, выберите этот пункт.

![Страница с настройками организации.](images/setup-organization-first-step.png)

> Остаётся согласиться с условиями с помощью чекбокса рядом с согласием на условия использования сервиса и можно жать на кнопку «Next» под формой. На следующей можете выбрать и подключить членов команды. Для этого в поле «Search by username, full name or email address» введите ник, имя или почту нужного участника. После нажмите на кнопку «Complete setup» под формой.

Reviewer 2: ```suggestion
Остаётся согласиться с условиями с помощью чекбокса рядом с «I hereby accept the Terms of Service...» и можно жать на кнопку «Next» под формой. На следующем шаге можете выбрать и подключить членов команды. Для этого в поле «Search by username, full name or email address» введите ник, имя или почту нужного участника. После нажмите на кнопку «Complete setup» под формой.
```

Согласиться с помощью согласия... Добавила текст лейбла цитатой, мб так лучше?
